topic/machine-learning/classification-automl/requirements.txt

```
@@ -0,0 +1,7 @@
+crate[sqlalchemy]==0.34.0
+mlflow-cratedb
```

Let this be driven by Dependabot.
```suggestion
mlflow-cratedb==2.7.1
```

---

@amotl Done. Please note that all the other requirements.txt files contain unfixed mlflow-cratedb package references. I'd vote for having consistency within the repo. Let me know if you want to contract this to me.